backend/app/data_migrations/populate_sector.py

```diff
@@ -0,0 +1,31 @@
+import csv
+
+from sqlalchemy.orm import Session
+
+from app.db.models import Sector
+from .utils import has_rows
+
+
+def populate_sector(db: Session) -> None:
```

Comment:
this is exactly the sort of thing we need, but let's talk about making the sector data independent of pre-existing id/parent_id fields.